sdk/src/client/node_api/core/routes.rs

```diff
@@ -138,8 +138,8 @@ impl Client {
     ) -> Result<ValidatorsResponse, ClientError> {
         const PATH: &str = "api/core/v3/validators";
         let query = query_tuples_to_query_string([
-            page_size.into().map(|i| ("pageSize", i.to_string())),
-            cursor.into().map(|i| ("cursor", i)),
+            page_size.into().map(|page_size| ("pageSize", page_size.to_string())),
+            cursor.into().map(|cursor| ("cursor", cursor)),
```

Comment:
Don't do this if you're not doing it in the whole file

Reply:
I would personally not do this change as we use single letter closures a lot, but if you really prefer, do it in the whole file, no need to be inconsistent

Reply:
> Don't do this if you're not doing it in the whole file

But doing that for the whole file would bloat the PR with unrelated changes depending on the number of unrelated occurences. 

> I would personally not do this change as we use single letter closures a lot, but if you really prefer, do it in the whole file, no need to be inconsistent

But that is not why I made the change: while `i` commonly represents `index` like in other places of this file, it does not usually represent `size` or a `cursor`. I don't mind single letter closures at all, but SUGGESTED this little change. I am fine with using `n` and `c` instead, or reverting completelty. But what am I supposed to do? Keep it a draft as long as I make suggetions??

Also ... your comment sounded a bit salty for no good reason. Do you want people to make suggestions, or not? Sometimes they necessarily won't be to your liking, but I have no problem with reverting the change.